tests/models/xapi/test_video.py

```diff
@@ -42,6 +42,23 @@ def test_models_xapi_video_selectors_with_valid_statements(class_, data):
     model = ModelSelector(module="ralph.models.xapi").get_first_model(statement)
     assert model is class_
 
+@pytest.mark.parametrize(
+    "class_",
+    [
+        VideoVolumeChangeInteraction,
+        VideoEnableClosedCaptioning, 
+        VideoScreenChangeInteraction,
+    ],
+)
+@custom_given(st.data())
+def test_models_xapi_video_selectors_with_valid_interaction_statements(class_, data):
+    """Tests given a valid video interaction xAPI statement the `get_models`
+    selector method should return the list of all interaction models.
+    """
+
+    statement = json.loads(data.draw(custom_builds(class_)).json())
+    models = ModelSelector(module="ralph.models.xapi").get_models(statement)
+    assert class_ in models
```

Comment:
@SergioSim not sure of my logic for this test ! Could you tell me what you think of that ?

Reply:
Indeed, that's better to use the validator. I'll fix the test